tests/integration/components/imgix-image-test.js

```diff
@@ -1,85 +1,126 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
-/* global URI */
-
 moduleForComponent('imgix-image', 'Integration | Component | imgix image', {
   integration: true
 });
 
 test('it renders', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
+  assert.ok(this.$());
+});
 
-  assert.equal(this.$().text().trim(), '');
-  assert.equal(this.$('img').length, 1);
+test('it renders event more better', function(assert) {
+  this.render(
+    hbs`<div style='width:200px;height:200px;'>{{imgix-image path='/users/1.png' }}</div>`
+  );
+
+  let url = new URL(this.$('img').attr('src'));
+  assert.equal(
+    this.$()
+      .text()
+      .trim(),
+    ''
+  );
+  assert.equal(url.pathname, '/users/1.png');
 });
 
 test('it renders the correct path', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
 
-  assert.ok(this.$('img').attr('src').indexOf("https://assets.imgix.net/users/1.png") > -1);
-  assert.ok(this.$('img').attr('src').indexOf("w=1280") > -1);
+  assert.ok(
+    this.$('img')
+      .attr('src')
+      .indexOf('https://assets.imgix.net/users/1.png') > -1
+  );
+  assert.ok(
+    this.$('img')
+      .attr('src')
+      .indexOf('w=1250') > -1
+  );
 });
 
 test('it builds the default URL', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
-  let url = URI(this.$('img').attr('src'));
+  let url = new URL(this.$('img').attr('src'));
 
-  assert.equal(url.search(true).w, "1280");
-  assert.equal(url.pathname(), "/users/1.png");
-  assert.equal(url.search(true).fit, "crop");
-  assert.equal(url.search(true).crop, "faces");
+  assert.equal(url.searchParams.get('w'), '1250');
```

Comment:
I'd love to see this test broken up into smaller units:
- "it builds a URL with the correct path"
- "it builds a URL with the expected width as a query parameter"
- "it builds a URL with the default `fit` value as a query parameter"
- etc.